### ISSUE TITLE
Insufficient filtering in hciconfig

### ISSUE THREAD

**Issue by bircoph** [linuxhw/hw-probe#96]
Hi!

hciconfig contains custom visible device name, which should be filtered, e.g.:
```
    Link mode: SLAVE ACCEPT
    Name: '<my bluetooth access point name>'
    Class: 0x00010c
```
I suggest using `'...'` as in other filtering places.

**Comment by linuxhw** [comment]
This is already fixed by commit c8a19268576a8f1c20279a185443ca87c1a5f8fd.

Thank you.

**Comment by bircoph** [comment]
Nice! Please consider a new release: it looks like many changes are accumulated on top of the latest one now.
